### DISCUSSION TITLE
Looking for tflite model for openai whisper

### DISCUSSION THREAD

**Discussion by nyadla-sys** [openai/whisper#193] [General]
Has anyone attempted to convert the OpenAI Whisper Pytorch model into the Tflite model?

**Comment by nyadla-sys** [comment]
Please use below colab to generate tflite models 
https://colab.research.google.com/github/usefulsensors/openai-whisper/blob/main/notebooks/tflite_from_huggingface_whisper.ipynb

tflite models:
https://github.com/usefulsensors/openai-whisper/tree/main/models

Follow README to know more details about whisper.tflite 
https://github.com/usefulsensors/openai-whisper

**Comment by nyadla-sys** [comment]
Please feel free to download the openai/whisper-tiny tflite-based Android APP from Google App Store.
https://play.google.com/store/apps/details?id=com.whisper.android.tflitecpp

**Reply by appsoft124** [reply]
> Please feel free to download the openai/whisper-tiny tflite-based Android APP from Google App Store. https://play.google.com/store/apps/details?id=com.whisper.android.tflitecpp

please send me your email i have paid task for you please share your email

**Comment by nyadla-sys** [comment]
Refer the below github repo for Android example, which uses whisper-tiny.en.tflite model 
https://github.com/nyadla-sys/whisper.tflite
